vcell-cli/src/main/java/org/vcell/cli/sbml/MemoryLogger.java

```diff
@@ -31,6 +32,7 @@ public void sendMessage(Priority p, ErrorType et, String message)
             lowPriority.add(msg);
         }
         System.err.println(p + " " + et + ": "  + message);
+        throw new VCLoggerException(message);
```

Comment:
Here I specifically wanted to only log problems and keep going (without throwing exception).

Reply:
.. and definitely wouldn't want to throw an exception for medium or low priority messages.